spec/fixtures/basic_custom_measurement.csv

```diff
@@ -0,0 +1,7 @@
+measurement_event,entity,name,measurement,value,
```

Comment:
@jtu0 Down the road we can consider expanding this to allow multiple `value`s to be specified to make creating bulk measurements cleaner and easier. Looking at White that may be highly desirable, but I don't think it's necessary for delivering the MVP to Pinto. Remind me to bring up this discussion when I meet with the stakeholders for White